samples/DocumentTaskExample/Program.cs

```diff
@@ -0,0 +1,175 @@
+
+                // This id MUST be unique in the comments part.
+                new CommentRangeStart() { Id = strCommentId },
+                new Run(
+                    new Text(strParagraphText)),
+                new CommentRangeEnd() { Id = strCommentId },
+                new Run(new CommentReference() { Id = strCommentId }))
+
+                // These ids MUST be unique in the document.
+                { TextId = "FFFFFF02" });
+        }
+
+        private static void AddMentionComment(MainDocumentPart mdp, string strCommentId, string mention, string commentText, User mentioner, User mentionee)
+        {
+            mdp.AddNewPart<WordprocessingCommentsPart>();
+            mdp.WordprocessingCommentsPart.Comments = new Comments();
+            Comments comments = mdp.WordprocessingCommentsPart.Comments;
+
+            comments.AppendChild<Comment>(
+                new Comment(
+                    new Paragraph(
+                        new Run(
+                            new RunProperties(
+                                new Color() { Val = "2B579A" },
+                                new Shading() { Val = ShadingPatternValues.Clear, Color = "auto", Fill = "E6E6E6" }),
+                            new FieldChar() { FieldCharType = FieldCharValues.Begin }),
+                        new Run(
+                            new FieldCode(" HYPERLINK \"mailto:" + mentionee.Email + "\"")
+                            { Space = SpaceProcessingModeValues.Preserve }),
+                        new BookmarkStart() { Id = "2", Name = "_@_0FD9AD1E39C946AB9F9E1352162C9910Z" },
+                        new Run(
+                            new RunProperties(
+                                new Color() { Val = "2B579A" },
+                                new Shading() { Val = ShadingPatternValues.Clear, Color = "auto", Fill = "E6E6E6" }),
+                            new FieldChar() { FieldCharType = FieldCharValues.Separate }),
+                        new BookmarkEnd() { Id = "2" },
+                        new Run(
+                            new RunProperties(
+                                new RunStyle() { Val = "Mention" },
+                                new NoProof()),
+                            new Text(mention)),
+                        new Run(
+                            new RunProperties(
+                                new Color() { Val = "2B579A" },
+                                new Shading() { Val = ShadingPatternValues.Clear, Color = "auto", Fill = "E6E6E6" }),
+                            new FieldChar()
+                            { FieldCharType = FieldCharValues.End }),
+                        new Run(
+                            new Text(commentText) { Space = SpaceProcessingModeValues.Preserve }),
+                        new Run(
+                            new AnnotationReferenceMark()))
+
+                    // These ids MUST be unique in the document.
+                    { TextId = "FFFFFF04" })
```

Comment:
Removed

Reply:
@mikeebowen, you said "Removed" but the `TextId` property is still set. If you set that property, you must also set the `ParagraphId` property. It would make the example easier to understand if you did not set `TextId`.